dbms/src/Storages/DeltaMerge/BitmapFilter/BitmapFilter.cpp

```diff
@@ -42,41 +42,39 @@ void BitmapFilter::set(BlockInputStreamPtr & stream)
     stream->readSuffix();
 }
 
-void BitmapFilter::set(const ColumnPtr & col, const FilterPtr & f)
+void BitmapFilter::set(const ColumnPtr & row_ids_col, const FilterPtr & f)
 {
-    const auto * v = toColumnVectorDataPtr<UInt32>(col);
-    set(v->data(), v->size(), f);
+    const auto * v = toColumnVectorDataPtr<UInt32>(row_ids_col);
+    set({v->data(), v->size()}, f);
 }
```

Comment:
this function is redundant, we can get it merged into `BitmapFilter::set(BlockInputStreamPtr & stream)` like this https://github.com/Lloyd-Pottiger/tiflash/pull/3